en/docs/administer/monitoring-server-health.md

```diff
@@ -28,39 +28,54 @@ The **Carbon Health Check API** can be used to check the health of WSO2 Identity
 
 ### Configuring the API
 
-This feature is disabled by default. To enable the API, set the
-`         <Enable>        ` property in the
-`         health-check-config.xml        ` file to **true**.  
+This feature is disabled by default. To enable the API, add following
+configuration to `deployment.toml` file in the [IS_HOME]/repository/conf
+location.
+
+```toml
+[carbon_health_check]
+enable= true
+```
 
 !!! tip
     
     If the feature has not been enabled successfully, a request to the API
     will only return a 200 OK response.
     
+There are two health-checkers that will get enabled when you enable
+health checkers using above global configuration. 
+    - DataSource Health Checker 
+    - User Store Health Checker
 
-**Sample health-check-config.xml file**
+**DataSource Health Checker**
 
-``` java
-<CarbonHealthCheckConfigs>
-    <Enable>true</Enable>
-    <HealthCheckers>
-        <HealthChecker name="DataSourceHealthChecker" orderId="97" enable="true">
-            <!--<Property name="monitored.datasources">jdbc/WSO2CarbonDB,jdbc/WSO2MetricsDB,jdbc/WSO2UMDB</Property>-->
-            <Property name="pool.usage.limit.percentage">80</Property>
-        </HealthChecker>
-        <HealthChecker name="SuperTenantUSHealthChecker" orderId="98" enable="true">
-            <!--<Property name="monitored.user.stores">primary,sec</Property>-->
-        </HealthChecker>
-    </HealthCheckers>
-</CarbonHealthCheckConfigs>
+Gives you the status about the datasources available in the system. You
+can configure the datasources to be monitored as below in
+`deployment.toml` file.
```

Comment:
Shall we modify this to "This health checker returns the status of the datasources available in the system. You can configure the datasources to be monitored by adding the following configuration to the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder.

Reply:
fixed